src/views/menus/NotificationsMenu.tsx

```diff
@@ -125,7 +133,16 @@ export const NotificationsMenu = ({
       isOpen={isMenuOpen || placement === DialogPlacement.Inline}
       setIsOpen={setIsMenuOpen}
       items={items}
-      title={stringGetter({ key: STRING_KEYS.NOTIFICATIONS })}
+      title={
+        <div tw="flex items-center justify-between">
+          {stringGetter({ key: STRING_KEYS.NOTIFICATIONS })}{' '}
+          <IconButton
+            iconName={IconName.Gear}
+            onClick={() => dispatch(openDialog(DialogTypes.Preferences()))}
+            fullIcon
+          />
+        </div>
+      }
```

Comment:
nit: (existing bug) when I click outside of the preferences dialog on mobile web to dismiss, it auto opens the dialog that pops up from clicking on the notif


https://github.com/user-attachments/assets/e15e4f25-f3d0-42ea-8cfc-6c8b33a64a27